packages/lib/server-only/user/get-all-users.ts

```diff
@@ -0,0 +1,71 @@
+import { Prisma } from '@prisma/client';
+
+import { prisma } from '@documenso/prisma';
+
+type getAllUsersProps = {
```

Comment:
Is there a reason this is camel case instead of pascal case?

Reply:
good catch. thanks!